### PR TITLE
Add form validation for login

### PR DESCRIPTION
**Pull request by axellorreyne**

### CODE REVIEW

File: src/popup/popup.js [hunk 1]
```diff
@@ -36,7 +36,14 @@ function submitLoginOnKeyEnter(event) {
 /**
  * Handle user's request to log in
  */
-async function handleOnClickLogin() {
+async function handleOnClickLogin(event) {
+    event.preventDefault();
+
+    let $form = document.getElementById("credential-input-forms");
```
Comment (pheyvaer) [inline src/popup/popup.js:42]: Can't this be `const`?

File: src/popup/popup.js [hunk 2]
```diff
@@ -96,11 +103,11 @@ function handleAfterLoginAttempt(options) {
         document.getElementById("login-status-success").classList.remove('hidden');
         document.getElementById("login-status-fail").classList.add('hidden');
         document.getElementById("login-button").classList.add("d-none");
-        document.getElementById('logout-button').classList.remove("d-none");
+        document.getElementById('logout-button-div').classList.remove("d-none");
     } else {
         document.getElementById('login-fail-error').innerText = error + '.';
         document.getElementById("login-status-fail").classList.remove('hidden');
-        document.getElementById("login-status-success").classList.add('hidden');
+        document.getElementById("login-status-success").classList.add('hiddenF');
```
Comment (pheyvaer) [inline src/popup/popup.js:110]: `hiddenF` instead of `hidden`. Can this even work? 😄

File: src/popup/popup.js (hunk 1 above)
Comment (axellorreyne) [reply src/popup/popup.js]: Changed in 81501da5f1fde73623a60ff35b9753d684d0a6f3

File: src/popup/popup.js (hunk 2 above)
Comment (axellorreyne) [reply src/popup/popup.js]: This was a typo. Fixed in 94c86e9db649a52cd9d0c518b064deede7942af3

**Review by pheyvaer — APPROVED** [review]
Tested locally and it works 🚀

**Pull request merged**